GUI/Dialogs/ManageGameInstancesDialog.cs

```diff
@@ -213,7 +213,7 @@ private void GameInstancesListView_Click(object sender, MouseEventArgs e)
 
         private void OpenDirectoryMenuItem_Click(object sender, EventArgs e)
         {
-            string path = GameInstancesListView.SelectedItems[0].SubItems[2].Text;
+            string path = GameInstancesListView.SelectedItems[0].SubItems[3].Text;
```

Comment:
Oops! :facepalm:

Reply:
Can't blame you for that one, hardcoding it like that is guaranteed to break at some point.
Thinking about it, how about recalculating it like this? No dependency on column count or ordering.
```
string path = _manager.Instances[(string) GameInstancesListView.SelectedItems[0].Tag].GameDir();
```

Reply:
I like that idea. :+1:

Reply:
Oops, that should've been up one line...